librubyfmt/src/format_prism.rs

```diff
@@ -737,8 +737,7 @@ fn format_class_node(ps: &mut dyn ConcreteParserState, class_node: prism::ClassN
     );
 
     if let Some(superclass) = class_node.superclass() {
-        ps.emit_ident("<".to_string());
-        ps.emit_space();
+        ps.emit_ident(" < ".to_string());
```

Comment:
This was just a silly thing from before I had implemented enough stuff for the tests with superclasses -- this is just adding the missing space before the `<`